File: engine/src/main/java/com/bartlomiejpluta/base/engine/audio/asset/SoundAsset.java
package com.bartlomiejpluta.base.engine.audio.asset;

import com.bartlomiejpluta.base.engine.common.asset.Asset;
import lombok.NonNull;

public class SoundAsset extends Asset {
   public SoundAsset(@NonNull String uid, @NonNull String source) {
      super(uid, source);
   }
}

File: editor/src/main/kotlin/com/bartlomiejpluta/base/editor/audio/asset/SoundAsset.kt
package com.bartlomiejpluta.base.editor.audio.asset

import com.bartlomiejpluta.base.editor.asset.model.Asset
import com.bartlomiejpluta.base.editor.project.model.Project

class SoundAsset(project: Project, uid: String, source: String, name: String) :
   Asset(project.audioDirectoryProperty, uid, source, name)